torchtune/datasets/_stack_exchange_paired.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Dict, Optional
+
+from torchtune.data import (
+    ChosenRejectedToMessages,
+    PromptTemplate,
+    QuestionAnswerTemplate,
+)
+from torchtune.datasets._preference import PreferenceDataset
+from torchtune.modules.tokenizers import ModelTokenizer
+
+
+def stack_exchange_paired_dataset(
+    tokenizer: ModelTokenizer,
```

Comment:
Why is this `tokenizer: ModelTokenizer`, but all other builders use `model_transform: Transform`?